openmc/geometry.py

```diff
@@ -23,6 +23,8 @@ class Geometry:
     root : openmc.UniverseBase or Iterable of openmc.Cell, optional
         Root universe which contains all others, or an iterable of cells that
         should be used to create a root universe.
+    **kwargs : dict, optional
+        Any keyword arguments are used to set attributes on the instance.
```

Comment:
For two parameters, this almost doesn't feel like it's worth it over just explicitly adding `merge_surfaces` and `surface_precision`

Reply:
I believe @eepeterson was keen on getting all the classes fully made from constructors as part of the __repr__, __str__ and then xml to python code idea. But admittedly I could have misunderstood

Reply:
Sorry if I was not clear. I meant rather than adding the catch-all `**kwargs` as an argument, explicitly add `merge_surfaces` and `surface_precision` arguments. So @eepeterson's goal can still be attained.

Reply:
Ah sorry, you were clear I just miss read. I changed to explicitly adding these args

Reply:
Thanks for starting this @shimwell! To follow on to @paulromano's comment: generally I think `**kwargs` should be used for pass through arguments to the superclass if necessary, whereas explicit keyword arguments are best for the class itself (especially if they are small in number).